python/src/main.cpp

```diff
@@ -1,333 +1,557 @@
+#include <pybind11/numpy.h>
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
-#include <pybind11/numpy.h>
```

Comment:
I'm not sure if this file shoudl also be encompassed by the formatter as it's very much structured to "circumvent" the shortcomings of the very cryptic ways in which pybind11 has to define the functions - mainly as adding the formatter could result in much less intuitive legibility on the already cryptic, so thinking we can skip this one as well

Reply:
Yes, this one should not be formatted. Reverted it.